command/ca/provisioner/webhook/update.go

```diff
@@ -0,0 +1,162 @@
+package webhook
+
+import (
+	"errors"
+
+	"github.com/smallstep/certificates/ca"
+	"github.com/smallstep/cli/flags"
+	"github.com/urfave/cli"
+	"go.step.sm/cli-utils/errs"
+	"go.step.sm/linkedca"
+)
+
+func updateCommand() cli.Command {
+	return cli.Command{
+		Name:   "update",
+		Action: cli.ActionFunc(updateAction),
+		Usage:  "update a webhook attached to a provisioner",
+		UsageText: `**step ca provisioner webhook update** <name> **--provisioner**=<name>
+[**--url**=<url>] [**--kind**=<kind>] [**--bearer-token**=<token>]
+[**--basic-auth-username**=<username>] [**--basic-auth-password**=<password>]
+[**--disable-tls-client-auth**]  [**--cert-type**=<cert-type>]
+[**--admin-cert**=<file>] [**--admin-key**=<file>] [**--admin-provisioner**=<name>]
+[**--admin-subject**=<subject>] [**--password-file**=<file>] [**--ca-url**=<uri>]
+[**--root**=<file>] [**--context**=<name>] [**--ca-config**=<file>]`,
+		Flags: []cli.Flag{
+			// General webhook flags
+			provisionerFlag,
+			urlFlag,
+			kindFlag,
+			bearerTokenFlag,
+			basicAuthUsernameFlag,
+			basicAuthPasswordFlag,
+			disableTLSClientAuthFlag,
+			certTypeFlag,
+
+			flags.AdminCert,
+			flags.AdminKey,
+			flags.AdminProvisioner,
+			flags.AdminSubject,
+			flags.PasswordFile,
+			flags.CaURL,
+			flags.Root,
+			flags.Context,
+			flags.CaConfig,
+		},
+		Description: `**step ca provisioner webhook update** updates a webhook attached to a provisioner.
+
+## POSITIONAL ARGUMENTS
+
+<name>
+: The name of the webhook.
+
+## EXAMPLES
+
+Change a webhook's url:
+'''
+step ca provisioner webhook update my_webhook --provisioner my_provisioner --url https://example.com
+'''
+
+Configure a webhook to send a bearer token to the server:
+'''
+step ca provisioner webhook update my_webhook --provisioner my_provisioner --bearer-token abc123xyz
+'''
+
+Change the password sent to the webhook with basic authentication:
```

Comment:
Everywhere else that we accept a password we generally do this through a file. Should this be `basic-auth-password-file`?

Reply:
https://github.com/smallstep/cli/commit/7963486e8d134b70d447490ea3ae3b1d7ec45c8f

Also changed bearer-token to read from file